### ISSUE TITLE
Test for UNUSED fails (of all things)

### ISSUE THREAD

**Issue by scotws** [scotws/TaliForth2#73]
This is probably some stupid one-off error: `{ here unused + 400 + -> 7FFF } INCORRECT RESULT: { here unused + 400 + -> 7FFF } ACTUAL RESULT: { -8000 } ok`

**Comment by SamCoVT** [comment]
In my mind, the answer Tali is coming up with (8000, if printed with u.) is the correct answer.  

Ignoring the $400 for a moment:
If you were at $7000, there are $1000 usable bytes left before $8000 ($7000 to $7fff).  If you add the location of the fist usable byte and the number of usable bytes, you will get the location of the first unusable byte.

I suggest either changing the expected answer to 8000 or changing the +400 to +3ff if you want to calculate the address of the last byte of ram.

**Comment by scotws** [comment]
Yeah, that makes sense. I'll change +400 to +3FF next chance I get.
